api_mt/integrations/index.md

```diff
@@ -0,0 +1,73 @@
+# __How to connect MTs to CAT tools__  
+
+
+> ___&ldquo;The biggest challenge is to combine technology and the human side without losing the latter&rdquo;___ — Gordana Antonijević  
+
+
+
+
+
+# Introduction  
+Machine Translation (MT) technology allows you to automatically translate your text from one language to another. Both Trados Studio 2021™ and memoQ translator pro 9.8™ use plugins to connect to machine translation engine vendors. This manual provides information as to how to obtain API keys from MT vendors and enter them into the plugins. 
+
+## MT vendors described  
+This manual shows how to obtain APIs to the following MTs:
+
+| API | Language pairs | Customization | Setup | Cost | Privacy |
+| ----------- | ----------- | ----------- | ----------- | ----------- | ----------- |
+| Google Cloud Translation Basic (Google Translate API v2)  | 100+  | Translation memory and glossary  | Difficult | $20 per 1M characters | No-trace, ISO 27001 and GDPR-compliant |
+| DeepL  |  26 | Glossary  | Easy | EUR 8.99 per user/5 files a month EUR 29.99 per user/20 files EUR 59.99 per user/100 files | No-trace, ISO 27001 and GDPR-compliant |
+| Amazon Translate (AWS)   |  75 | Glossaries, Adaptive customization, to and from English only  | Difficult | $15 per 1 M characters, $60 for adaptive customization per 1 M characters | Amazon shared responsibility model, ISO 27001-compliant |
+| Modern MT   |  40+ | Glossaries, Adaptive customization for all pairs  | Easy | $25 per month for professionals/unlimited no. of characters, $50 for real-time adaptive | No-trace, ISO 27001 and GDPR-compliant |
+
+# Privacy
+More information on how the MT vendors listed in [MT vendors described](#mt-vendors-described) process your data is available from:
```

Comment:
We try not to do this kind of sentence completion thing, because it's hard to translate or machine-translate.

Reply:
Edited as @bittlingmayer suggested. I love this level of translation and localization awareness!!!